coopr-docs/docs/source/guide/superadmin/plugins.rst

```diff
@@ -155,7 +155,21 @@ The only required return value is ``status``, where ``'status': 0`` represents s
 represents failure. A raised exception will also result in failure.
 
 Additionally, your provider plugin will likely need to return information such as a machine's ID, SSH credentials and
-public IP, so that it can be used in subsequent tasks. For these cases, simply write the results as key-value pairs underneath
+public IP, so that it can be used in subsequent tasks.  Special facilities exist for reporting the machine's hostname and
```

Comment:
Only a single space is required after a period. (There's another instance on line 159.)